aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
I don't want to say middleware because it's not about the reverse proxy component that we need to install. It needs to be very clear that there's something missing from the _application_.
The last part was particularly hard to debug (a few hrs lost!) since: 
  a. it wasn't mentioned anywhere in the doc and 
  b. the doc points out that most of the time it's the reverse proxy middleware's fault if the app is working without IIS.